### PR TITLE
Clear `OperationError` before attempting to checkout PR

### PR DESCRIPTION
**Pull request by jcansdale**
Fix for #786.

### CODE REVIEW

**Comment by grokys** [comment]
Seems the change from `Exception` to `FileNotFoundException` made some other tests fail!

**Comment by jcansdale** [comment]
LOL. Oops! I'll fix it.

**Comment by jcansdale** [comment]
Seem to be working now...

**Comment by grokys** [comment]
Thinking about this, we should probably also clear the error on refresh?

**Comment by jcansdale** [comment]
@grokys I'm now clearing `OperationError` on refresh (`Initialize`) and also before Pull and Push.

File: src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs [hunk 1]
```diff
@@ -137,6 +137,12 @@ public class PullRequestDetailViewModel : BaseViewModel, IPullRequestDetailViewM
             DiffFile = ReactiveCommand.Create();
         }
 
+        void SubscribeOperationError(ReactiveCommand<Unit> command)
```
Comment (grokys) [inline src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs]: Small nit: could you put this with the other private methods down below the public methods?

**Comment by jcansdale** [comment]
@grokys Done. 😄

**Pull request merged**